ecosystem/typescript/sdk/src/providers/provider.ts

```diff
@@ -28,6 +28,8 @@ export class Provider {
 
   indexerClient: IndexerClient;
 
+  network: Network | "CUSTOM";
```

Comment:
Could you define this as a new constant first? Like make a new type:
```
type NetworkWithCustom = Network | "CUSTOM"
```

That kind of thing.